how-we-work-together.md

```diff
@@ -174,69 +174,69 @@ Many Sparkboxers are involved in initial staffing, budget, and timeline conversa
 
     * Design animation/interactions early in project
 
-    * Let PMs know status regularly and if more time needed as early as possible
+    * Let PMs know status regularly and if more time is needed as early as possible
 
-    * Give valuable, honest design review feedback in design reviews
+    * Give valuable, honest feedback in design reviews
 
-### We Present Concepts Intentionally 
+### We Present Concepts Intentionally
 
-* Participate in the design review before concept is presented to client
+* Participate in the design review before the concept is presented to client
 
     * Give valuable feedback in a respectful way
 
-    * All team members serve to make the project better
+    * Remember that all team members serve to make the project better
 
-* Ensure before presenting to the client that the concept is buildable in time and budget
+* Ensure the concept is buildable within the timeline and budget before presenting it to the client
 
-    * Make sure to involve the right people who can give input on the effort to build a design concept
+    * Involve the right people who can give input on the effort to build a design concept
 
-* Present design concepts to the client with a detailed explanation and reasons behind visual choices
+* Present design concepts to the client with a detailed explanation of reasons behind visual choices
 
     * Talk about the why, not the what
 
     * Reiterate the visual vocabulary
 
-    * Discuss how the presented concept will help to accomplish the project goals
+    * Discuss how the presented concept will help accomplish the project goals
 
-    * If the client doesn’t agree with our rational, we listen and take every effort to address their concerns
+    * If the client doesn’t agree with our rational, we listen and make every effort to address their concerns
 
-    * We believe it is acceptable and our job to respectfully push back when we feel strongly that a client suggestion is detrimental to the project, and give reasons why
+    * We believe it is acceptable and our job to respectfully push back and give reasons why when we feel strongly that a client suggestion is detrimental to the project
 
-* Communicate/educate the client that end goal is browser
+* Communicate to/educate the client that the end goal is browser
 
-    * Different browsers render fonts, colors differently
+    * Different browsers render fonts and colors differently
 
     * May not show comps of every page

```

Comment:
I'm not quite sure what this is saying. Will we know present comps of every page? Will the browser not render comps of every page? How does this relate to the browser?

Reply:
I only edited this page to remove Zenhub.